### PR TITLE
Added contestant number (order) for WSContest contest page

### PR DESCRIPTION
**Pull request by Avartii**
I have added contestant number (order) for WSContest contest page. It will be beneficial to the contestant as well as admins to be able to tell, for example: who are the top 10 contestant, how many total contestants are there, etc.

### CODE REVIEW

**Review by Aditya0545 — APPROVED** [review]

**Comment by Avartii** [comment]
Hello, @samwilson will you please review this commit and provide me feedback for the same.

File: templates/contests_view.html.twig [hunk 1]
```diff
@@ -55,6 +55,7 @@
             <caption style="caption-side: top">{{ msg('scores-legend', [score_calculation_interval~'']) }}</caption>
             <thead>
                 <tr>
+                    <th>{{ msg('rank') }}</th>
```
Comment (samwilson) [inline templates/contests_view.html.twig:58]: The `rank` message needs to be added to `i18n/en.json` and `i18n/qqq.json`.

**Review by samwilson — CHANGES_REQUESTED** [review]
Thanks for this! Just one small thing to fix, then it looks like it's ready to go.

Did you have any troubles getting this set up to work on and test locally?

**Comment by Avartii** [comment]
> Thanks for this! Just one small thing to fix, then it looks like it's ready to go.
> 
> Did you have any troubles getting this set up to work on and test locally?

No, I'm not having any issues when testing locally; I just assumed it was a minor edit and so didn't checked it thoroughly.

File: templates/contests_view.html.twig (hunk 1 above)
Comment (Avartii) [reply templates/contests_view.html.twig:58]: Okay I got your point, submitting the updated patch very soon! Thanks for your guidance.

**Comment by samwilson** [comment]
> No, I'm not having any issues when testing locally; I just assumed it was a minor edit and so didn't checked it thoroughly.

Okay, no worries.

It looks like you've created #73 as a replacement for this PR, is that right? Let's close this one and continue the discussion over there. (In general, you don't need to create separate PRs for fixing review issues, you can just push more commits to the branch here.)

**Pull request closed without merging**